### PR TITLE
Update json to 20240205

### PR DESCRIPTION
**Pull request by wendigo**
No release notes needed

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -457,7 +457,7 @@
             <dependency>
                 <groupId>com.clickhouse</groupId>
                 <artifactId>clickhouse-jdbc</artifactId>
-                <version>0.5.0</version>
```
Comment (martint) [inline pom.xml:460]: Are there any compatibility changes with this update?

Comment (wendigo) [reply pom.xml:460]: We test against:

- 21.8.14.5 (EOL 21 Aug 2022)
- 21.11.10.1
- 21.8.13.1.altinitystable
- 20.8.4.11_aes

Coverage didn't change

Comment (wendigo) [reply pom.xml:460]: Does it answer your concern @martint ?

Comment (martint) [reply pom.xml:460]: Yup, thanks.

**Review by martint — APPROVED** [review]

**Pull request merged**